components/infobox/wikis/starcraft2/infobox_skill_ability.lua

```diff
@@ -186,31 +186,28 @@ end
 function Ability:getCostDisplay()
```

Comment:
Ability:getCostDisplay
CustomBuilding:_getCostDisplay
Spell:getResearchCost
CustomUnit:_getCostDisplay

does things differently for the same thing

Reply:
imo oos for this pr
please do not resolve this comment so that i can look into it tomorrow when on pc (probably in a sep. PR)

Reply:
#2963